src/test.ts

```diff
@@ -438,6 +438,13 @@ const performRequest = async (requestObject: requestsObjectSchema, requestName:
     }
   }
 
+  if (typeof requestObject.request.json !== 'undefined') {
```

Comment:
```suggestion
  if (requestObject.request.json !== undefined) {
```

It's the same but cleaner

Reply:
Agree, I literally copied it from the previous if to keep the same code style but I'm happy to change it.